public/v4/apps/invoiceninja.yml

```diff
@@ -0,0 +1,125 @@
+                - RUN curl -o /etc/nginx/conf.d/in-vhost.conf https://raw.githubusercontent.com/invoiceninja/dockerfiles/$$cap_app_tag/config/nginx/in-vhost.conf
+                - RUN sed -i "s/app:9000/srv-captain--$$cap_appname-php:9000/g" /etc/nginx/conf.d/in-vhost.conf
+                - ENTRYPOINT ["/docker-entrypoint.sh"]
+                - EXPOSE 80
+                - CMD mkdir -p /var/www/app/docker-backup-public && nginx -g 'daemon off;'
+
+    $$cap_appname-db:
+        image: mariadb:$$cap_mariadb_tag
+        networks: invoiceninja
+        restart: always
+        volumes:
+            - $$cap_appname-db-data:/var/lib/mysql
+        environment:
+            MYSQL_ROOT_PASSWORD: $$cap_db_root_pass
+            MYSQL_USER: $$cap_db_user
+            MYSQL_PASSWORD: $$cap_db_pass
+            MYSQL_DATABASE: $$cap_db_name
+        caproverExtra:
+            notExposeAsWebApp: 'true'
+
+caproverOneClickApp:
+    variables:
+        - id: $$cap_app_tag
+          label: Invoice Ninja Tag
+          defaultValue: '5.1.53'
+          description: 'Check out their docker page for the valid tags https://hub.docker.com/r/invoiceninja/invoiceninja/tags'
+        - id: $$cap_user_email
+          label: User Email
+          defaultValue: user@example.com
+        - id: $$cap_user_password
+          label: User Password
+          defaultValue: $$cap_gen_random_hex(8)
+        - id: $$cap_app_key
+          label: App Key
+          defaultValue: $$cap_gen_random_hex(32)
+        - id: $$cap_app_debug
+          label: App Debug
+          defaultValue: 'true'
+        - id: $$cap_phantomjs_pdf_generation
+          label: PhantomJS PDF Generation
+          defaultValue: 'false'
+        - id: $$cap_phantomjs_cloud_key
+          label: PhantomJS Cloud Key
+          defaultValue: 'a-demo-key-with-low-quota-per-ip-address'
+          description: 'https://phantomjscloud.com/'
+        - id: $$cap_nginx_tag
+          label: Nginx Tag
+          defaultValue: '1.20.0-alpine'
+          description: 'Check out their docker page for the valid tags https://hub.docker.com/_/nginx?tab=tags'
+        - id: $$cap_mariadb_tag
+          label: MariaDB Version
+          defaultValue: 10.4
+          description: 'Check out their docker page for the valid tags https://hub.docker.com/_/mariadb?tab=tags'
+        - id: $$cap_db_root_pass
+          label: MariaDB Root Password
+          defaultValue: $$cap_gen_random_hex(32)
+        - id: $$cap_db_user
+          label: MariaDB Database User
+          defaultValue: 'ninja'
+        - id: $$cap_db_name
+          label: MariaDB Database Name
+          defaultValue: 'ninja'
+        - id: $$cap_db_pass
+          label: MariaDB Database Password
+          defaultValue: $$cap_gen_random_hex(32)
+    instructions:
+        start: >-
+            Invoice Clients, Track Work-Time, Get Paid Online.
+            For more info visit https://www.invoiceninja.com/
```

Comment:
Would you be able to provide a link that explains the environmental variables such as `PHANTOMJS_PDF_GENERATION` and etc... ?

Reply:
Dig really hard to find this out https://github.com/invoiceninja/invoiceninja/issues/3390#issuecomment-797291670

Reply:
I have just updated Caprover var's name for the self-explained reason.